Source/santad/EventProviders/AuthResultCache.mm

```diff
@@ -128,12 +128,10 @@ static inline uint64_t TimestampFromCachedValue(uint64_t cachedValue) {
 
     // SNTActionHoldAllowed and SNTActionHoldDenied are used for transitions, however the
     // cached action is translated to SNTActionRespondAllow or SNTActionRespondDeny respectively.
+    // We do not want to cache this result and later execs need to go through this path again.
     case SNTActionHoldAllowed: OS_FALLTHROUGH;
-    case SNTActionHoldDenied:
-      return cache->set(vnode_id,
-                        CacheableAction(decision == SNTActionHoldAllowed ? SNTActionRespondAllow
-                                                                         : SNTActionRespondDeny),
-                        CacheableAction(SNTActionRespondHold, 0));
+    case SNTActionHoldDenied: cache->remove(vnode_id); return YES;
```

Comment:
To confirm my understanding - previously, when this result was only used during Standalone Mode, we would be cache responses. But now since we want to ensure CEL TouchID expressions will always get authorized, we simply remove the placeholder from the cache. For users in standalone mode without a CEL TouchID rule, this will result in Santa performing one extra rule lookup on the traditional path before getting into the cache.